projects/storefrontapp-e2e-cypress/cypress/helpers/login.ts

```diff
@@ -44,13 +50,21 @@ export function loginWithBadCredentials() {
 }
 
 export function loginAsDefaultUser() {
+  const loginPage = waitForPage('/login', 'getLoginPage');
   cy.get(loginLinkSelector).click();
+  cy.wait(`@${loginPage}`).its('status').should('eq', 200);
+
   login(defaultUser.name, defaultUser.password);
 }
 
-export function listenForTokenRevocationReqest(): string {
+export function listenForTokenRevocationRequest(stub = false): string {
   const aliasName = 'tokenRevocation';
   cy.server();
-  cy.route('POST', '/authorizationserver/oauth/revoke').as(aliasName);
+
+  if (stub) {
```

Comment:
I don't think we need to stub those requests. We only need to drop assertions for 200 status on token revocation waits.